chromium_src/media/audio/pulse/pulse_util.cc

```diff
@@ -0,0 +1,16 @@
+// Copyright (c) 2020 The Brave Authors
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// you can obtain one at http://mozilla.org/MPL/2.0/.
+
+#include "build/branding_buildflags.h"
+
+#if defined(OFFICIAL_BUILD)
```

Comment:
what is this needed for? We don't set GOOGLE_CHROME_BRANDING to true and the default is false